xrootd not built with krb5

Leading to errors on lxplus such as:

```console
xrdcp output_test.root  [root://eospublic.cern.ch//eos/experiment/fcc/ee/tmp/test1.root](https://mmm.cern.ch/owa/UrlBlockedError.aspx)
Plugin version SecClnt v5.3.2 is incompatible with seckrb5 v5.4.2 (must be <= 5.3.x) in sec.protocol libXrdSeckrb5-5.so
220419 17:24:34 14342 cryptossl_X509CreateProxy: EEC certificate has expired
[0B/0B][100%][==================================================][0B/s]  
Run: [ERROR] Server responded with an error: [3010] Unable to open file /eos/experiment/fcc/ee/tmp/test1.root; Operation not permitted (destination)
```

--- a/packages/key4hep-stack/package.py
+++ b/packages/key4hep-stack/package.py
@@ -69,6 +69,7 @@ class Key4hepStack(BundlePackage, Key4hepPackage):
     depends_on('bhlumi')
     depends_on('babayaga')
     depends_on('delphes')
+    depends_on('xrootd +krb5')
 
     ##################### general purpose generators ######
     #######################################################

